tests/instances.spec.ts

```diff
@@ -228,3 +230,21 @@ test("instance yaml edit", async ({ page }) => {
 
   await deleteInstance(page, instance);
 });
+
+test("instance terminal operations", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await visitAndStartInstance(page, instance);
+  await page.getByTestId("tab-link-Terminal").click();
+  await expect(page.locator(".xterm-screen")).toBeVisible(TIMEOUT);
+  const terminalInput = page.getByRole("textbox", { name: "Terminal input" });
+  await terminalInput.fill("lsb_release -a");
+  await terminalInput.press("Enter");
+  await expect(page.locator(".xterm-rows")).toContainText("jammy");
```

Comment:
This is going to change for sure. Maybe check for the text "Distributor ID:", or something else that is not expected to change?

Reply:
Changed the text check to `Ubuntu`. Although in `createInstance` we always select a `jammy` image, will we update that to the next lts version released?

Reply:
Yeah I think for this specific test using Ubuntu should be fine :+1: